README.md

```diff
@@ -4,6 +4,9 @@ This repo contains the source code and documentation powering [bn.react.dev](htt
 
 ## Getting started
 
+### Join the Bengali discord community
+Hi dev, We are Bengali React doc translation community welcome you to join the discord server - https://github.com/reactjs/bn.react.dev/issues/26
```

Comment:
should have been the server link?

Reply:
Bhai I try to keep the issue as the main point for community link and update. That's why I added the issue intentionally. Like for progress we follow one issues.

Reply:
Oh! now I got it. I guess in that case we could make the code inline and a bit paraphrase like this?

Hi dev, we are Bengali React doc translation community. We welcome you to [join our discord server.](https://github.com/reactjs/bn.react.dev/issues/26)

Reply:
Also, I'd suggest making the line in the issue inline as well. Like...
Anyway, approving the PR.

Here is the discord server invitation [link.]( https://discord.gg/6YG28yBF)

Reply:
Ahh! I thought approving wouldn't just merge it immediately. I am so sorry.